### ISSUE TITLE
How to install the 1.01 version?

### ISSUE THREAD

**Issue by MelvinMik** [akatz-ai/ComfyUI-DepthCrafter-Nodes#11]
I dont get how to install and use the 1.01 model, caus from the huggingface the last updated model is 3 months old that cant be it?

**Comment by akatz-ai** [comment]
Hi @MelvinMik, the 1.0.1 update uses the same model, the only things that change are the default inference configuration. I was also confused when I first looked into their code but that's all that changed.

**Comment by MelvinMik** [comment]
Hey so if you update the nodes you already work with the 1.01 version?

**Comment by akatz-ai** [comment]
Yes the latest custom node update has the inference configuration set to 1.0.1 specified defaults
